requirements.txt

```diff
@@ -1,2 +1,2 @@
 pbr>=2.0
-sphinx>=1.5,<2.2
+sphinx>=1.5,<=2.2
```

Comment:
You should bump this to `2.3` too, otherwise we won't be able to use `2.2.1`, right?

Reply:
So <2.3 ?

Reply:
From reading the build failure I'd say you will fix the build by setting it to `<2.3`.

Reply:
Correct, `<2.3`